src/api/general-services/pipeline-manage/constructors/create-new-step.js

```diff
@@ -0,0 +1,91 @@
+const config = require('../../../../config');
+
+const createNewStep = (context, step, args) => {
```

Comment:
I see that this is not used anywhere at the moment -- are we going to start using it after the UI bits for QC are implemented too?

Reply:
It's used in `pipeline-manage/index.js`,